features/step_definitions/ability_steps.rb

```diff
@@ -0,0 +1,69 @@
+Given("I have created several character abilities") do
+  FactoryBot.create(:character,
+                    :user => @registered_user,
+                    :name => "Trol",
+                    :description => "My first trol")
+
+end
+
+When("I visit character show page") do
+  visit user_path(@registered_user)
+  click_on "Trol"
+  click_on "New Ability"
+  fill_in "ability_name", :with => "Power"
+  fill_in "ability_value", :with => "10"
+  click_on "Create Ability"
+  click_on "New Ability"
+  fill_in "ability_name", :with => "Speed"
+  fill_in "ability_value", :with => "15"
+  click_on "Create Ability"
+end
```

Comment:
In this step, you create 2 abilities, but step title says  "I visit character show page"